src/main/groovy/net/wasdev/wlp/gradle/plugins/utils/LooseApplication.groovy

```diff
@@ -47,9 +47,9 @@ public class LooseApplication {
         }
     }
 
-    public void addManifestFile(Element parent, Task task, String pluginId) throws Exception {
+    public void addManifestFile(Element parent, Project project, String pluginId) throws Exception {
```

Comment:
Same here because I don't see any code changes to methods that are calling these existing methods.

Reply:
never mind.